src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -384,7 +384,8 @@ INTEGRATED_CHANNELS_API_CHUNK_TRANSMISSION_LIMIT:
 JWT_EXPIRATION: 30
 LANGUAGE_CODE: en
 LANGUAGE_COOKIE: {{ env "ENVIRONMENT" }}-openedx-language-preference
-LEARN_AI_API_URL: https://{{ key "edxapp/learn-ai-api-domain" }}  # Added for ol_openedx_chat
+MIT_LEARN_AI_API_URL: https://{{ key "edxapp/learn-ai-api-domain" }}  # Added for ol_openedx_chat
+MIT_LEARN_API_URL: https://{{ key "edxapp/mit-learn-api-domain" }}  # Added for ol_openedx_chat
```

Comment:
@blarghmatey I am not sure about the key for the learn API, I just added this here for reference. Could you please update this so that it points to the Learn API. For RC the Learn URL is `https://api.rc.learn.mit.edu`